### DISCUSSION TITLE
Electron with Eartrumpet

### DISCUSSION THREAD

**Discussion by chinarui-na** [File-New-Project/EarTrumpet#801] [Q&A] [answered]
I like Eartrumpet very much . Now I want to control the volume of some programs myself in certain situations.  i use nodejs and electron. Is there some api i can use? Thank you very much.

**Comment by davuxcom** [comment, marked as answer]
EarTrumpet isn't an API that can be consumed. It could be loaded and objects used but you would also need to bridge .net into electron and it's probably just not worth it.

**Reply by riverar** [reply]
Tonight we'll merge in EarTrumpet Actions work thus far into dev. So if you're interested in automation, key bindings, etc. install an experimental dev build! (See https://eartrumpet.app for a link.)

**Reply by D4n2021** [reply]
@riverar why is the dev build not released to stable yet, do some users still experience crashes that need fixing?
